pxr/imaging/hdSt/materialXFilter.cpp

```diff
@@ -634,26 +634,48 @@ _GetGlTFSurfaceMaterialTag(HdMaterialNode2 const& terminal)
     return materialToken.GetString();
 }
 
-static const mx::TypeDesc*
+static const mx::TypeDesc
 _GetMxTypeDescription(std::string const& typeName)
 {
+#if MATERIALX_MAJOR_VERSION == 1 && MATERIALX_MINOR_VERSION==38
+  // Add whatever is necessary for current codebase:
+  static const auto _typeLibrary =
```

Comment:
Minor thing, but I wonder if you could have an ifdef around
"using typename MxTypeDesc = const mx::TypeDesc"
or something and then only define the map once, since aside from the map type everything is character-for-character identical.  You'd still need another ifdef around the find/return code below, but that's less likely to be edited than this type map table.

Reply:
yep - seems like a good improvement.